.appveyor.yml

```diff
@@ -0,0 +1,24 @@
+build: false
+
+environment:
+  global:
+      REQUIREMENTS: "numpy scipy scikit-learn pytest flake8"
+
+  matrix:
+    - PYTHON_VERSION: 3.6
+      MINICONDA: C:\Miniconda3-x64
+
+
+init:
+  - "ECHO %PYTHON_VERSION% %MINICONDA%"
+
+install:
+  - "set PATH=%MINICONDA%;%MINICONDA%\\Scripts;%PATH%"
+  - conda config --set always_yes yes --set changeps1 no
+  - conda create -n sklego-env -c pytorch %REQUIREMENTS% python=%PYTHON_VERSION%
```

Comment:
Is there a particular reason pytorch is in there?

Reply:
wayyyyy in the beginning I must've added that. removed it for now.